### DISCUSSION TITLE
2024.03.14(목)

### DISCUSSION THREAD

**Discussion by Crack-Egg** [Party-Time-Job/Party-Time-Job#50] [회의록]
## 1. 공통 컴포넌트

작업을 진행하면서 필요한 부분은 계속 추가해 나가는 방향으로 진행
(대신 기존에 필요한 기능들은 변경하지 않기!)

## 2. 데이터

시간날 때 Swagger에서 데이터 쌓기!

## 3. 쿠키 사용

우선은 localStorge로 사용하고, 추후에 기능 완성되고 리팩토링 진행하면서 수정 예정

## 4. 배포하기

Vercel로 배포할 예정!

**Comment by DanteSnow** [comment]
추가 내용
1. axios 사용 관련 규격화
- 오늘 재명님이 올리신 PR에 맞게 작업
